### PR TITLE
doc/install: rethink install doc installation methods order

### PR DESCRIPTION
**Pull request by zdover23**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
jenkins render docs

File: doc/install/index.rst [hunk 1]
```diff
@@ -1,72 +1,108 @@
...
-	Install Ceph Object Gateway <install-ceph-gateway>
-	Install Virtualization for Block <install-vm-cloud>
+Installs Ceph using containers and systemd.
 
+* :ref:`cephadm-bootstrap`
 
-Deploy a Cluster Manually
-=========================
+     * cephadm is supported only on Octopus and newer releases.
+     * cephadm is fully integrated with the new orcehstration API and fully supports the new CLI and dashboard features to manage cluster deployment.
```
Comment (liewegas) [inline doc/install/index.rst]: typo

**Comment by liewegas** [comment]
- let's reorder these in index.rst:
```
   install/index
   cephadm/index
   start/index
```
so that this comes *before* cephadm and ceph-deploy?

- also, can we rename start/ to ceph-deploy/ maybe?  (probably need to search/replace in the rest of the tree to make sure there aren't direct links to start/)

**Comment by sebastian-philipp** [comment]
> * also, can we rename start/ to ceph-deploy/ maybe?  (probably need to search/replace in the rest of the tree to make sure there aren't direct links to start/)

Can we do this in a separate PR?

File: doc/install/index.rst [hunk 2]
```diff
@@ -1,72 +1,108 @@
...
+     * cephadm requires container support (podman or docker) and Python 3.
```
Comment (sebastian-philipp) [inline doc/install/index.rst]:
```suggestion
     * cephadm requires container support (podman or docker).
```
I'd omit "Python 3" here, as this is mostly an implementation detail

Comment (liewegas) [reply doc/install/index.rst]: :+1:

**Comment by sebastian-philipp** [comment]
jenkins retest this please

File: doc/install/index.rst (hunk 1 above)
Comment (zdover23) [reply doc/install/index.rst]: note to future Zac: s/orcehstration/orchestration/

**Pull request merged**